src/main/resources/site/processors/add-metadata.js

```diff
@@ -9,8 +9,14 @@ const view = resolve("add-metadata.html");
 
 exports.responseProcessor = function (req, res) {
     const site = libs.portal.getSite();
-    const content = libs.portal.getContent();
     const siteConfig = libs.common.getTheConfig(site);
```

Comment:
Onrelated, but I must comment.
this does not look like a correct way to get site config
There is a method `portalLib.getSiteConfig`.

Reply:
This was created so it will work without context

Reply:
But `getSite()` may return null, when there are no sufficient rights on it, but sufficient rights on content. So, this code is broken.